src/main/java/life/qbic/io/commandline/OpenBISPasswordParser.java

```diff
@@ -22,22 +17,30 @@ public static String readPasswordFromConsole() {
     return String.valueOf(passwordChars);
   }
 
+  /**
+   *
+   * @param variableName commandline variable that might be null or empty
+   * @return
+   */
+  public static Boolean isNotNullOrEmpty(String variableName) {
+    Boolean NotNullOrEmpty = false;
+    if (variableName != null && !variableName.isEmpty()) {
+      NotNullOrEmpty = true;
+    }
+    return NotNullOrEmpty;
```

Comment:
```suggestion
    Boolean notNullOrEmpty = false;
    if (variableName != null && !variableName.isEmpty()) {
      notNullOrEmpty = true;
    }
    return notNullOrEmpty;
```
😉

Reply:
I like this method. As you do not use it within the parser itself you could think about moving it to the app class itself.

Reply:
Thank you for this suggestion, I was not sure where to put this method and it does make more sense in the app class itself! I'll change that 😄